.github/workflows/linux-snap.yml

```diff
@@ -1,11 +1,6 @@
 name: Linux Snap
 
 on:
-  push:
-    paths-ignore:
-    - scripts/docker/**
-    branches:
-    - master
```

Comment:
Why would this NOT run on push to master - that's how we usually trigger the builds...
So I would have thought the other way around... don't run on pull!

Reply:
It _will_ build on `master`, but in https://snapcraft.io/subsurface/builds, for all the architectures together - and publish to the `edge` channel. And for `current` and `snap-stable`, it will build [on Launchpad](https://launchpad.net/subsurface/+snaps). That's how it was until now (modulo `current` -> `candidate`, which is new).

I mean, we _could_ build here, but only for amd64 - for any other architecture we'd need to send it to Launchpad anyway (or qemu... slooow), so I thought to focus on that - it's a better infrastructure for everything snap :)

So the actions workflow I've left for PR validation, but published builds come from LP.

Reply:
Sorry about my confusion. Makes sense